perftest/simulations/README.md

```diff
@@ -90,6 +90,37 @@ Important note: The Gatling `Session` object is _immutable_! This means, when yo
 to the `Session`, you receive a cloned object, which needs to be passed downstream in your code or,
 as in most cases, returned form the actions' code.
 
+## Running a single Nessie Gatling simulation
+
+Gatling's Gradle plugin recognizes tasks in the form `gatlingRun-<fqcn>`, where `<fqcn>` is a dot-separated package-and-classname.  For instance, to run `KeyListSpillingSimulation` from this directory, execute:
+
+```
+../../gradlew gatlingRun-org.projectnessie.perftest.gatling.KeyListSpillingSimulation
```

Comment:
Nit: this can be run from the project directory using
```
./gradlew :nessie-perftest-simulations:gatlingRun-org.projectnessie.perftest.gatling.KeyListSpillingSimulation
```

Reply:
Applied (on this line, and the others like it in the PR)